### DISCUSSION TITLE
Which version to use for features: on next button click and dynamic fields on checkbox

### DISCUSSION THREAD

**Discussion by venkatesh1146** [manojadams/metaforms-core#73] [Q&A]
Hi @manojadams 

We fixed mui-forms v1.0.27 as the checkbox with dynamic fields is working. 
But while testing some of the functionalities are not working in other form.
[Here is the sample project with the test schema replicating the issue](https://github.com/venkatesh1146/mui-forms-test-project)

It contains both new schema [healthy - checkbox issue] and the old one [fourwheeler - next button not working]. 
Kindly check.

Thank you.

**Comment by manojadams** [comment]
@venkatesh1146 
Disabled sections are now not accessible through buttons. If we want to only disable tab navigation, it can be done through 
config param (changes TODO). Can you suggest a parameter? For example
```
<MuiForms 
    config={{
      tabs: {
        navigationDisabled: [2,3]
      }  
    }}
    schema={schema}
    onSubmit={(data) => {
        console.log(data);
    }}ta, allData) => {
        console.log(formData, allData);
    }}
    onSubmit={(data) => {
        console.log(data);
    }}
/>
```

**Reply by venkateshpullaganti** [reply]
`Disabled sections are now not accessible through buttons.`
Means even after filling all the fields in the first section/tab ??

**Comment by manojadams** [comment]
@venkatesh1146 @venkateshpullaganti Sorry for the breaking changes in `mui-forms`. 
Suggesting the following 2 step approach to disable tabs .
- Remove isDisabled key from json schema for section fields.
- Add disabled tabs info in config

Let me know what do you think?
```
<MuiForms 
    config={{
      tabs: {
        disabled: [2,3]
      }  
    }}
    schema={schema}
    onSubmit={(data) => {
        console.log(data);
    }}ta, allData) => {
        console.log(formData, allData);
    }}
    onSubmit={(data) => {
        console.log(data);
    }}
/>
```

**Reply by venkatesh1146** [reply]
Okey @manojadams 
So do I need to update the package version??
